webpack/globalizeDecode.ts

```diff
@@ -0,0 +1,12 @@
+import * as decode from "@decode/clarity";
+import * as clarity from "@src/clarity";
+
+// When built with webpack for prod, compiled clarity-js bundle doesn't expose the module anywhere on the page.
+// Since we need clarity-js having the decode functionality to be available globally, we can create a wrapper
+// module that would assign clarity to window.
+if (typeof window !== "undefined") {
+    if ((window as any).clarity === undefined) {
+        (window as any).clarity = clarity;
```

Comment:
There won’t be a valid clarity variable. So the follow line would throw an error since we will attempt to assign a property “decode” of a null object. Can you please change this to { decode: decode } and the line below should be in else clause.